packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,352 @@
+    </Item>
+  )}
+</ListView>
+```
+
+### Internationalization
+To internationalize a ListView, all text content within the ListView should be localized. This includes the `aria-label` provided to the ListView if any.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of ListView is automatically flipped.
+
+## Labeling
+### Accessibility
+An `aria-label` must be provided to the ListView for accessibility. If the ListView is labeled by a separate element, an `aria-labelledby` prop must be provided using the id of the labeling element instead.
+
+## Asynchronous loading
+ListView supports loading data asynchronously, and will display a progress circle reflecting the current load state,
+set by the `loadingState` prop. It also supports infinite scrolling to load more data on demand as the user scrolls, via the `onLoadMore` prop.
+
+This example uses the [useAsyncList](../react-stately/useAsyncList.html) hook to handle loading the data.
+See the docs for more information.
+
+```tsx example
+import {useAsyncList} from '@react-stately/data';
+
+function AsyncList() {
+
+  let list = useAsyncList({
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      let res = await fetch(cursor || `https://swapi.py4e.com/api/people/?search=`, {signal});
+      let json = await res.json();
+
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  return (
+    <ListView
+      selectionMode="multiple"
+      aria-label="example async loading list"
+      maxWidth="size-6000"
+      height="size-3000"
+      items={list.items}
+      loadingState={list.loadingState}
+      onLoadMore={list.loadMore}>
+      {(item) => (
+        <Item key={item.name} textValue={item.name}>{item.name}</Item>
+      )}
+    </ListView>
+  );
+}
+```
+
+## Complex items
+
+Items within a ListView also allow for additional content used to add context or provide additional actions to items. Icons, links, and descriptions can be added to the children of `<Item>` as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements. Additionally, components such as `<ActionButton>`, `<ActionGroup>`, and `<ActionMenu>` will be styled appropriately if included within an item.
+
+```tsx example
+<ListView selectionMode="multiple" maxWidth="size-6000" aria-label="List example with complex items">
+  <Item textValue="Row with link, icon, and description">
+    <Folder />
+    <Link>Row with link, icon, and description</Link>
+    <Text slot="description">Description of row</Text>
+  </Item>
+  <Item textValue="Row with action button">
+    Row with action button
+    <ActionButton aria-label="Info">
+      <Info />
+    </ActionButton>
+  </Item>
+  <Item textValue="Row with action group">
+    Row with action group
+    <ActionGroup buttonLabelBehavior="hide">
```

Comment:
IMO I think this is a different case since ListView is a single column only. Unlike TableView you won't get trapped within the action group

Reply:
I was concerned because this used a role of grid instead of list. Isn't the issue with multiple interactives is there are within a gridview/datagrid.

Reply:
My understanding of this was that we disallow the action group in the TableView since the keyboard navigation was messed up, not so much that it was disallowed within the grid aria pattern (I may be mistaken about this though). For the TableView case we were previously looking into a grid edit mode to solve this.

Reply:
Yeah, this should be fine for ListView because of the reason @LFDanLu described.